GVRf/Framework/framework/src/main/java/org/gearvrf/GVRMesh.java

```diff
@@ -362,14 +362,6 @@ public GVRMesh getBoundingBox() {
         return mBones;
     }
     /**
```

Comment:
now any if any of the mesh changes, batching will be disabled for that mesh. so there is no more "dynamic"/ "static" now. same applies for material/ renderdata.